### DISCUSSION TITLE
How do i remove my domain from this eth blacklist?

### DISCUSSION THREAD

**Discussion by momoduoladapo** [MetaMask/eth-phishing-detect#41082] [Q&A]
Please can someone tell me how they were able to get this resolved on their domain?

![image](https://github.com/MetaMask/eth-phishing-detect/assets/5340964/01c4c42f-2b27-4e44-83aa-09aaafa6e027)

Your answers will be highly appreciated.

**Comment by momoduoladapo** [comment]
Thanks but do you have any suggestions?

**Reply by mustafabucak01** [reply]
Anti-phishing feature is based on user reports.

It's likely because fraudsters have reported your website just to annoy you or cause harm to you, and to undermine the integrity of MetaMask's antiphishing features.

The only way to fix the issue is to contact MetaMask antiphishing team and wait them to manually fix the issue.
https://ethereum.stackexchange.com/questions/139814/metamask-incorrectly-flagged-a-website-as-phishing-site
https://kinsta.com/blog/deceptive-site-ahead/

I found them.
